include/utils/DetailViewUtilsTest.php

```diff
@@ -252,4 +252,137 @@ public function testgetRelatedListInfoById() {
 		);
 		$this->assertEquals($expected, $relationInfo, 'testgetRelatedListInfoById');
 	}
-}
\ No newline at end of file
+
+	/**
+	 * Method testgetRelatedListInfoById
+	 * @test
+	 */
+	 public function testgetRelatedModulesInfomation() {
+		$currentModule = 'Accounts';
+		$focus = CRMEntity::getInstance($currentModule);
+		$restrictedRelations = array();
+		$actual = getRelatedModulesInfomation($currentModule);
```

Comment:
You are missing the $user variable here, once you add it to coreBOS pass in here the admin user. Then pass in that same user in the Assets test below.